scripts/ci/lib.sh

```diff
@@ -1223,9 +1223,17 @@ store_test_results() {
     if ! is_in_PR_context; then
     {
         info "Creating JIRA task for failures found in $from"
```

Comment:
what about wrapping the {} with a set +u, set -u? so we avoid any undefined causing a script exit?

Reply:
Or even better, just explicitly provide defaults in each variable invocation. We might be able to come up with better values than empty strings at least in some cases.

Reply:
I'm not sure if explicit defaults are better as they are harder to read and empty string is good enough in our case

Reply:
I don't know whether I agree with the "good enough" part, but meh..